src/utils/__tests__/fixtures/caterpillars.json

```diff
@@ -28,7 +28,7 @@
         "OBJECTID": 308778,
         "GlobalID": "218b8c15-ef75-4851-9616-125132af7438",
         "BoomID": "584073",
-        "Registratie": "",
+        "Registratie": "Melding",
```

Comment:
Can we have the possible statuses in a type?

Reply:
goog point

Reply:
in the meta definition